### PR TITLE
Do not display non essential tf commands

### PR DESCRIPTION
**Pull request by msarvar**

### CODE REVIEW

**Review by nishkrishnan — APPROVED** [review]

File: server/handlers/project_command_output_handler.go [hunk 1]
```diff
@@ -167,11 +167,6 @@ func (p *AsyncProjectCommandOutputHandler) writeLogLine(pull string, line string
 	}
 	p.receiverBuffersLock.Unlock()
 
-	// No need to write to projectOutputBuffers if clear msg.
-	if line == models.LogStreamingClearMsg {
-		return
-	}
-
```
Comment (Aayyush) [inline server/handlers/project_command_output_handler.go:174]: Do we want to write ClearMsg to the `projectOutputBuffers`?

**Review by Aayyush — APPROVED** [review]
Looks good except for the one comment I had!

File: server/handlers/project_command_output_handler.go (hunk 1 above)
Comment (msarvar) [reply server/handlers/project_command_output_handler.go]: I reverted it

**Review by Aayyush — APPROVED** [review]
lgtm

**Pull request merged**